_config.yml

```diff
@@ -110,17 +110,21 @@ RIMdev:
     avatar: https://avatars0.githubusercontent.com/u/5313708?s=460&v=4
     title: UI/UX Developer
     github: tedk13
+  Ryan Trimble:
+    avatar: https://avatars.githubusercontent.com/u/5217768?s=460?v=4
+    title: UI/UX Developer
+    github: mrtrimble
+  # Previous Team Members
```

Comment:
Do you want to move John to former just while you're in here?

Reply:
RIP @jpvicari